spark-connector/spark-common/src/test/java/org/apache/gravitino/spark/connector/integration/test/sql/SparkQueryRunner.java

```diff
@@ -83,6 +84,7 @@ public SparkQueryRunner(SparkTestConfig sparkTestConfig) {
     }
     initSparkEnv();
 
+    abstractIT = new AbstractIT();
```

Comment:
Why not use inheritance instead of a new instance?

Reply:
see below.